spec/support/requests_spec_helper.rb

```diff
@@ -8,7 +8,7 @@ def api(version = 0.0)
     "/api/v#{version}"
   end
 
-  def disable_tenancy
-    stub_const("ENV", "BYPASS_TENANCY" => "true")
+  def bypass_tenancy(&block)
```

Comment:
Blocks all the way down.  🐢 

I really like this approach, but you don't need `&block` unless you are referencing it.